Say Their Names/Source/Service/Service.swift

```diff
@@ -12,6 +12,7 @@ import UIKit
 final class Service {
     lazy private(set) var navigator = Navigator(service: self)
     lazy private(set) var image = ImageService()
+    lazy private(set) var dateFormatter = DateFormatterService.shared
```

Comment:
Lets keep this a single instance. We're trying to avoid any possible singletons.